pkg/clusteragent/clusterchecks/dispatcher_rebalance.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+const tolerationMargin float64 = 0.9
```

Comment:
can we add a comment above explaining what this is for, and that 0.9 is tentative, and we're monitoring/open to feedback if this should be changed?